app/components/assets/assets-index/asset-quick-actions.tsx

```diff
@@ -0,0 +1,98 @@
+import { CopyIcon, PencilIcon, QrCodeIcon, Trash2Icon } from "lucide-react";
+import { Button } from "~/components/shared/button";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from "~/components/shared/tooltip";
+import type { AssetsFromViewItem } from "~/modules/asset/types";
+import { tw } from "~/utils/tw";
+import { DeleteAsset } from "../delete-asset";
+import QrPreviewDialog from "../qr-preview-dialog";
+
+type AssetQuickActionsProps = {
+  className?: string;
+  style?: React.CSSProperties;
+  asset: Pick<AssetsFromViewItem, "id" | "title" | "mainImage"> & {
+    qrId: string;
+  };
+};
+
+export default function AssetQuickActions({
+  className,
+  style,
+  asset,
+}: AssetQuickActionsProps) {
+  return (
+    <div className={tw("flex items-center", className)} style={style}>
+      <Tooltip>
+        <TooltipTrigger asChild>
+          <Button
+            size="sm"
+            variant="block-link-gray"
+            to={`/assets/${asset.id}/edit`}
+          >
+            <PencilIcon className="size-4" />
+          </Button>
+        </TooltipTrigger>
+
+        <TooltipContent align="center" side="top">
+          Edit asset information
+        </TooltipContent>
+      </Tooltip>
+
+      <Tooltip>
+        <TooltipTrigger>
```

Comment:
Here are we are getting nested buttons. So we are getting button inside button. We have to figure out how to make it work with `asChild` and have the tooltip still work. You can check the browser console for the error.

Reply:
I have fixed this issue.